tests/unit_tests/c_wrappers/res/job_queue/test_job_queue_manager_torque.py

```diff
@@ -184,3 +204,32 @@ def test_run_torque_job(tmpdir, qsub_script, qstat_script):
 
     # The "done" callback:
     assert (runpath / "OK").read_text(encoding="utf-8") == "success"
+
+
+def test_that_torque_driver_passes_dash_x_to_qstat(
```

Comment:
To be honest  I need an explanation on this one

Reply:
done on video.